extern/Catch2/docs/test-fixtures.md

```diff
@@ -10,9 +10,9 @@ class UniqueTestsFixture {
   private:
    static int uniqueID;
   protected:
-   DBConnection conn;
+   DBPoint point;
```

Comment:
Revert all the changes in this file.

Reply:
Reverted all three of these. Must have forgotten to ignore them when I was searching.